services/motion/builtin/builtin_utils_test.go

```diff
@@ -190,7 +190,7 @@ func createMoveOnGlobeEnvironment(ctx context.Context, t *testing.T, origin *geo
 	injectedCamera := inject.NewCamera("injectedCamera")
 	cameraLink := referenceframe.NewLinkInFrame(
 		baseLink.Name(),
-		spatialmath.NewPoseFromPoint(r3.Vector{X: 1, Y: 0, Z: 0}),
+		spatialmath.NewPose(r3.Vector{X: 1}, &spatialmath.OrientationVectorDegrees{OZ: 1, Theta: 90}),
```

Comment:
We wanted to have the camera facing in the y forward direction so that it is detecting things in front of it.  Currently it is detecting things to its right